### DISCUSSION TITLE
How to best configure tracker to avoid digi-repeating beacons

### DISCUSSION THREAD

**Discussion by Peter-ols** [richonguzman/LoRa_APRS_iGate#189] [Q&A] [answered]
This might be a beginner's question, sorry about that.
The recommended setting for the "path" configuration parameter is "WIDE1-1". But in the area I presently operate in, there is no need for digi-repeating - (almost) everything gets picked up by iGates.
So, what is the best string to place in the path to avoid any digirepeaters to repeat the beacon packages on RF, and thus save some energy and channel time.
Best Regards, Peter

**Comment by richonguzman** [comment, marked as answer]
Hi Peter,

WIDE1-1 is the recommended minimal path for repetitions , so as you may guess, not having WIDE1-1 avoids any digirepeated packets.

so just don't add it and it won't be repeated. BUT: what if you move to another place? would you mod the tracker path beforehand?
I mean, maybe if no digi is available , it wont hurt to Tx a few more bytes but have a more robust posible packets path.

how many igates are at your place?

**Reply by Peter-ols** [reply]
Dear Richardo,
Thank you for that quick answer! (as well as all your work on the great software!). When I take the tracker to an unknown area I would also change the configuration path to WIDE1-1. I live in a suburban area  in a rather flat country, my own iGate often gates packets from car-mobile units at a distance of up to 30 km. Within that range there are already a mixture of 6-7 LoRa iGates, digirepeaters and even combined iGates and digirepeaters. I don't understand why some iGates simultaneously  operates as a digirepeater, so I appreciate the (currently experimental) setting for fall-back digirepeater function in the iGate software. I have turned it on in my otherwise "pure" iGate (but our internet access and WiFi is quite stable here, so it is difficult to test).
